license_sh/analyze/analyze_shared.py

```diff
@@ -131,3 +133,34 @@ def add_analyze_to_dep_tree(analyze_dict: Dict, dep_tree: AnyNode):
                 node_analyze.pop("file", None)
                 node.analyze.append(node_analyze)
     return dep_tree
+
+
+def transformHtml(htmlText: str, ignored_tags: List = IGNORED_HTML_TAGS) -> str:
```

Comment:
rename to `transform_html` See Naming styles in https://realpython.com/python-pep8/

Reply:
The same applies to `html_text`

Reply:
Ah, javascript side is too strong with me :D

Reply:
Fixed